libs/cluster/Session/ClusterCommands.cs

```diff
@@ -1025,20 +1021,18 @@ public bool ProcessSlotManageCommands(ReadOnlySpan<byte> bufSpan, ReadOnlySpan<b
                     // CLUSTER SETSLOTRANGE NODE <node-id> <slot-start> <slot-end> [slot-start slot-end]
                     // CLUSTER SETSLOTRANGE STABLE <slot-start> <slot-end> [slot-start slot-end]
 
-                    SlotState slotState;
                     string nodeid = default;
                     var _count = count - 1;
                     var ptr = recvBufferPtr + readHead;
                     // Extract subcommand
                     if (!RespReadUtils.ReadStringWithLengthHeader(out var subcommand, ref ptr, recvBufferPtr + bytesRead))
                         return false;
 
-                    // Parse slot state
-                    try { slotState = (SlotState)Enum.Parse(typeof(SlotState), subcommand); }
-                    catch (Exception ex)
+                    // Try parse slot state
+                    if (!Enum.TryParse(subcommand, out SlotState slotState))
                     {
                         // Log error for invalid slot state option
-                        logger?.LogError(ex, "");
+                        logger?.LogError($"The given {nameof(SlotState)} is invalid.");
```

Comment:
Let's give a little bit better error for the logs..
```suggestion
                        logger?.LogError("The given '{input}' is not a valid slot state option.", subcommand);
```